keps/sig-node/4188-kubelet-pod-readiness-api/README.md

```diff
@@ -66,76 +70,87 @@ Items marked with (R) are required *prior to targeting to a milestone / release*
 
 ## Summary
 
-Proposal to add a new Kubelet gRPC API with endpoint returning local Pods readiness information.
-Serving that information by Kubelet within a Node will increase reliability and reduce load to the Kubernetes API Server and traffic outside the node. A connectivity issue between Node and Control Plane should not impact workloads which depend on Pods readiness statuses.
+Proposal to add a new Kubelet gRPC API with endpoint returning local Pods information,
+including the full PodSpec and status, not just readiness.  
```

Comment:
```suggestion
including the full PodSpec and status.  
```

person who will read this KEP the first time should not know about the history of this proposal =)